umpleonline/ump/GeometricSystem.ump

```diff
@@ -115,143 +115,154 @@ class NonPolyhedra{
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}
 }
 
 class Cylinder{
 	isA NonPolyhedra;
 	double radius;
 	double height;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}	
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}	
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}
 }
 
 class Cone{
 	isA NonPolyhedra;
 	double radius;
 	double height;
 	double sideLength;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}
-	boolean isEqual(NonPolyhedra object){/*implementation */}
-	boolean isLessThan(NonPolyhedra object){/*implementation */}			
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(NonPolyhedra object){return true;}			
 }
 
 class Cube{
 	isA Polyhedra;
 	double edgeLength;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}		
-	boolean isEqual(Polyhedra object){/*implementation */}
-	boolean isLessThan(Polyhedra object){/*implementation */}	
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}		
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(Polyhedra object){return true;}	
+	boolean isLessThan(NonPolyhedra object){return true;}	
 }
 
 class RectangularPrism{
 	isA Polyhedra;	
 	double length;
 	double width;
 	double height;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}	
-	boolean isEqual(Polyhedra object){/*implementation */}
-	boolean isLessThan(Polyhedra object){/*implementation */}		
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}	
+	boolean isEqual(NonPolyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
+	boolean isLessThan(Polyhedra object){return true;}	
+	boolean isLessThan(NonPolyhedra object){return true;}		
 }
 
 class TriangularPyramid{
 	isA Polyhedra;
 	double slideLength;
 	double height;
-	double volume(){/*implementation */}
-	double surfaceArea(){/*implementation */}
-	boolean isEqual(Polyhedra object){/*implementation */}
-	boolean isLessThan(Polyhedra object){/*implementation */}			
+	double volume(){return 1.0;}
+	double surfaceArea(){return 1.0;}
+	boolean isEqual(Polyhedra object){return true;}
+	boolean isEqual(CurvedShape object){return true;}
```

Comment:
Looks like the following method should be deleted.
`boolean isEqual(CurvedShape object){return true;}`